crates/chat-cli/src/cli/chat/tool_manager.rs

```diff
@@ -795,6 +795,35 @@ impl Clone for ToolManager {
     }
 }
 
+#[cfg(windows)]
+/// In Windows, we need to configure the `execute_cmd` tool instead of `execute_bash`.
+fn configure_windows_cmd(tool_specs: &mut HashMap<String, ToolSpec>) {
+    use serde_json::json;
+
+    use crate::cli::chat::tools::InputSchema;
+
+    tool_specs.remove("execute_bash");
```

Comment:
Maybe it's better to just remove `execute_bash` from the JSON file and conditionally insert it in Unix rather than removing it.